src/domains/ecommerce-app/api/ecommerce-webview/v1/_base_policy.xml.tpl

```diff
@@ -18,10 +18,11 @@
     <validate-jwt header-name="Authorization" failed-validation-httpcode="401" failed-validation-error-message="Unauthorized" require-expiration-time="true" require-scheme="Bearer" require-signed-tokens="true" output-token-variable-name="jwtToken">
         <issuer-signing-keys>
             <key>{{ecommerce-webview-jwt-signing-key}}</key>
+            <key>{{ecommerce-checkout-transaction-jwt-signing-key}}</key>
```

Comment:
add TODO to document that this part will be reviewed following the technical debt

Reply:
Fixed with 2fbef2bf60fcb8c594eca7472d1fca9a17b8faec